src/App.js

```diff
@@ -1,21 +1,21 @@
-import React, { Component } from 'react';
+import React , { Component } from 'react';
```

Comment:
Extra space after 'React'. I've added basic eslint config to our project recently, so please check issues in codestyle that are found by eslint. Some of the can be ignored, but the major ones should be fixed.